scripts/convertConfig.ts

```diff
@@ -296,7 +296,9 @@ function convertConfig(dataDir: string, verbose: number) {
             if (verbose >= 3) console.log(`.... book: `, JSON.stringify(books[0]));
         }
         const collectionNameTags = tag.getElementsByTagName('book-collection-name');
-        const collectionName = collectionNameTags.length > 0 ? collectionNameTags[0].innerHTML : '';
+        const collectionName = collectionNameTags[0].innerHTML.length
+            ? collectionNameTags[0].innerHTML
+            : undefined;
```

Comment:
This will error if the collection name tag is completely missing instead of empty. Is it ever possible for the `<book-collection-name>` tag to be completely left out of the xml? 

Ditto on lines 318 and 323.